Public/Src/Tools/SymbolDaemon/SymbolDaemon.cs

```diff
@@ -321,14 +321,21 @@ public static void SerializeSymbolsMetadata(Dictionary<ContentHash, HashSet<Debu
                 for (int i = 0; i < hashCount; i++)
                 {
                     ContentHash.TryParse(reader.ReadLine(), out var hash);
+                    Contract.Assert(hash.HashType == HashType.Vso0);
+                    var blobIdentifier = new Microsoft.VisualStudio.Services.BlobStore.Common.BlobIdentifier(hash.ToHashByteArray());

```

Comment:
>Microsoft.VisualStudio.Services.BlobStore.Common. [](start = 45, length = 49)

Can it not be shortened by `using Microsoft.VisualStudio.Services.BlobStore.Common` ?

Reply:
no - the same class is defined in BuildXL.Cache.* and in Microsoft.VisualStudio.*; so I need a full name

---
In reply to: [336120401](https://github.com/microsoft/BuildXL/pull/1041#discussion_r336120401) [](ancestors = 336120401)